pylot/loggers/chauffeur_logger_operator.py

```diff
@@ -0,0 +1,175 @@
+from absl import flags
+from collections import deque
+
+import cv2
+import numpy as np
+import threading
+import carla
+import json
+
+# ERDOS specific imports.
+from erdos.op import Op
+from erdos.utils import setup_logging
+
+# Pylot specific imports.
+from pylot.perception.segmentation.utils import transform_to_cityscapes_palette
+import pylot.utils
+import pylot.simulation.carla_utils
+import PIL.Image as Image
+
+FLAGS = flags.FLAGS
```

Comment:
You don't need the global flags here because you're passing them to the __init__ method.